### PR TITLE
harvey: make move smarter

### PR DESCRIPTION
**Pull request by sevki**

### CODE REVIEW

File: targets/harvey/move.go [hunk 1]
```diff
@@ -25,7 +29,26 @@ func (m *Move) Build(c *build.Context) error {
 }
 
 func (m *Move) Installs() map[string]string {
-	return m.Exports
+	installs := make(map[string]string)
+	for _, d := range m.Dependencies {
+		u := url.Parse(d)
+
+		// Lest figure out where things go
```
Comment (rminnich) [inline targets/harvey/move.go]: Lets

**Comment by sevki** [comment]
@rminnich does this make sense to you?

**Comment by rminnich** [comment]
it doesn't make sense to me. 

I would have expected the tree you're building to match the layout of the source. that's how it's done on every other project I've done this sort of thing on. 

So, let's pretend BUILD_OUT is /tmp/x. if I run bldy I'd expect to see
/tmp/x/amd64/bin/ip/ipconfg
/tmp/x/amd64/lib/libc.a
/tmp/x/amd64/bin/cat
/tmp/x/sys/src/9/amd64/harvey.32bit

IOW, the tree you build should look just like the harvey tree without sources in it. It should be possible to do this:
rsync -av /tmp/x/ $HARVEY (note the trailing / in /tmp/x/ is intentional)

we should not use rsyncv, of course, but if we do use it, it should work.

Then all your special cases and maybe some commands would disappear.

**Comment by sevki** [comment]
I tried that but when you preserve their source tree locations ipconfig for instance ends up in `sys/src/cmd/ip/ipconfig/ipconfig` not `amd64/bin/ip/ipconfg`

**Comment by sevki** [comment]
I assume this is why you've done something similar here https://github.com/Harvey-OS/harvey/blob/master/sys/src/cmd/ip/ipconfig/ipconfig.json#L6

**Comment by sevki** [comment]
not necessary anymore

**Pull request closed without merging**